docs/getting-started/writing-tutorial.md

```diff
@@ -260,40 +260,19 @@ Now `ick run` shows a dry-run summary of the changes that would be made:
 <!-- [[[cog show_cmd("ick run") ]]] -->
 ```console
 $ ick run
--> move_isort_cfg FAIL
-     move_isort_cfg
-     isort.cfg +0-3
-     pyproject.toml +3-0
+-> move_isort_cfg OK
 ```
-<!-- [[[end]]] (sum: NMU6ekvaPt) -->
+<!-- [[[end]]] (sum: CQwDezy1z7) -->
 
 Passing the `--patch` option displays the full patch of the changes that would
 be made:
 
 <!-- [[[cog show_cmd("ick run --patch") ]]] -->
 ```console
 $ ick run --patch
--> move_isort_cfg FAIL
```

Comment:
I think these were broken before but are working correctly now that they're project-scoped?